mappings/net/minecraft/client/render/entity/model/AnimalModel.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_4592 net/minecraft/client/render/entity/model/AnimalModel
+	FIELD field_20915 isHeadScaled Z
```

Comment:
Is this true if the entity's head shrinks with scale? Also no `is` prefix for fields.

Reply:
it's true if it's scaled *somehow*. The scale is calculated with `1.5F / field_20918`